### DISCUSSION TITLE
Please make an option where we can long press to copy the bio

### DISCUSSION THREAD

**Discussion by leryeo** [dayanch96/BHTikTok-Plus#37] [General]
I can't find anything in the current settings that lets me do this. Often people put their social media handles in their bios so it will be a lot easier for me to add them if I can just copy the entire bio.

**Comment by dayanch96** [comment]
![IMG_5742](https://github.com/dayanch96/BHTikTok-Plus/assets/38832025/9711b283-127b-45cc-9dd5-a555cb3dd219)

**Comment by leryeo** [comment]
I enabled this but whenever I go to long press the bio it brings up the see translation interface. I went into language settings and disabled translation but the interface still pops up when I long press the bio? In this picture I am trying to copy the bio that includes the email address as an example. 
![IMG_1361](https://github.com/dayanch96/BHTikTok-Plus/assets/156037577/868580db-e532-4ed5-8c5f-36991f3a89cd)
